app/assets/javascripts/actions/story.js

```diff
@@ -26,27 +31,36 @@ export const editStory = (id, newAttributes) => ({
   newAttributes
 });
 
+export const setLoadingStory = (id, error) => ({
```

Comment:
This one seems wrong, we're not meant to have an error here, right? I think the error is supposed to be in `updateStoryFailure`.